libbeat/docs/upgrading.asciidoc

```diff
@@ -215,3 +216,39 @@ order of operations should be:
 If downtime is not acceptable, another possible approach is to configure a
 different index pattern in the new Beat version, but this will likely require
 adjustments to your Kibana dashboards.
+
+[[upgrade-5-5-dashboards]]
+=== Upgrade Kibana dashboards
+
+In previous versions of Beats, when you ran the `import_dashboards` command, the
+Beats dashboards were automatically loaded into the `.kibana` index in
+Elasticsearch. This design made it more difficult for Beats developers to
+provide dashboards that were compatible with each new version of Kibana.
+
+In version 5.6, the Beats dashboards are still loaded by default into the
+`.kibana` index in Elasticsearch, but support has been added for loading
+dashboards via the Kibana API. Regardless of how you import the dashboards
+in version 5.6, they will work when you upgrade to Kibana 6.0. 
+
+However, if you have Beats 5.5 dashboards and plan to use them with Kibana 6.0,
+you need to import them into 5.6 now by using the Kibana API, or they will no
+longer work in Kibana 6.0.
```

Comment:
I don't think this is true. Kibana automatically upgrades any dashboards that work in 5.6, so as long as the user follows the normal upgrade sequence, there shouldn't be any need for extra instructions.